test/TestUtils.kt

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright (C) 2022 Vaticle
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ *
+ */
+
+package com.vaticle.typedb.studio.test
+
+import androidx.compose.ui.test.junit4.ComposeContentTestRule
+import kotlinx.coroutines.runBlocking
+import java.nio.charset.StandardCharsets
+import java.nio.file.Files
+import java.nio.file.Paths
+
+fun runComposeRule(compose: ComposeContentTestRule, rule: suspend ComposeContentTestRule.() -> Unit) {
+    runBlocking { compose.rule() }
+}
+
+fun fileNameToString(fileName: String): String {
+    return Files.readAllLines(Paths.get(fileName), StandardCharsets.UTF_8)
+        .joinToString("")
+}
```

Comment:
We will need these simple patterns in testing everywhere, so having them available to all the tests rather than re-written is best practice.